### ISSUE TITLE
Clyde in DMs notification shows as user on the other end

### ISSUE THREAD

**Issue by ErrorOliver** [Aliucord/Aliucord#372]
### Discord Account

ErrorOliver

### What happens when the bug or crash occurs?

The bug happens then I/other user tries to ping Clyde in DMs and then Clyde responds, user isn't on DM conversation then the notification within discord will show as the user on the other end.

### What is the expected behaviour?

It should say Clyde within the users DMs.

### How do you recreate this bug or crash?

1. Other user pings Clyde while user is not in current DM conversation (Or quickly switch DM conversation then Clyde is pinged by you)
![Screenshot_20230725_141304](https://github.com/Aliucord/Aliucord/assets/100078292/2ecc8cba-19f6-4f90-83ec-a05dc3d8c885)
![Screenshot_20230725_141313](https://github.com/Aliucord/Aliucord/assets/100078292/8aaed719-d106-4047-b1cb-3dd0e19c4b11)

### Request Agreement

- [X] I did indeed check to make sure the bug or crash report is applicable!

**Comment by rushiiMachine** [comment]
> The bug happens then I/other user tries to ping Clyde in DMs and then Clyde responds, user isn't on DM conversation then the notification within discord will show as the user on the other end.

Please rephrase this to be comprehensible

**Comment by ErrorOliver** [comment]
Thanks @rowancze for helping me with this
> This bug occurs when someone within the DM conversation attempts to ping Clyde in DMs and then it responds. The notification is shown as if it came from the other person in the DM conversation even though it came from Clyde
